src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -2611,6 +2612,49 @@ public bool RelativeMotionUnGrab()
             return ret;
         }
 
+        /// <summary>
+        /// Sets the window blur using window blur information.
+        /// 
+        /// It is designed to apply a blur effect to a window based on specified parameters. 
+        /// This supports different types of blurring effects, including blurring the window's background only.
+        /// Or blurring the area surrounding the window while keeping the window itself clear.
+        /// The more information is written WindowBlurInfo struct.
+        /// </summary>
+        /// <param name="info"> Blur information for Window Blur.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetBlur(WindowBlurInfo info)
+        {
+            // Create Blur Info instance and binding.
+            IntPtr internalBlurInfo = Interop.WindowBlurInfo.New((int)info.BlurType, info.BlurRadius, info.BackgroundCornerRadius);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            Interop.Window.SetBlur(SwigCPtr, internalBlurInfo);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            Interop.WindowBlurInfo.DeleteWindowBlurInfo(internalBlurInfo);
+        }
+
+        /// <summary>
+        /// Gets the current window blur information.
+        /// </summary>
+        /// <returns>Returns current window blur information.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WindowBlurInfo GetBlur()
+        {
+            IntPtr internalBlurInfo = Interop.WindowBlurInfo.New();
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            
+            internalBlurInfo = Interop.Window.GetBlur(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            this.blurInfo.BlurType = (WindowBlurType)Interop.WindowBlurInfo.GetBlurType(internalBlurInfo);
```

Comment:
C#의 struct는 value타입으로 이렇게 하나를 만들어서 계속 사용하는데에 큰 잇점이 없습니다. 어차피 stack에만 존재하는 값이라서 그냥 여기서 새로 생성해서 리턴하는게 좋을듯합니다.

```suggestion
            var blurInfo = new WindowBlurInfo();
            blurInfo.BlurType = (WindowBlurType)Interop.WindowBlurInfo.GetBlurType(internalBlurInfo);
```

Reply:
새로 생성해서 return을 했던니, memory가 계속 증가해서 별도의 private field 로 두었습니다.

Reply:
다른 원인으로 증가한듯 합니다. c#에서 struct는 stack에만 존재합니다.
별도의 private field로 두었더니 증가하는게 멈췄나요?

Reply:
네, 증가하는 것이 멈추었습니다. 다시 new를 사용해서 메모리 증가 여부를 확인해 보겠습니다.

Reply:
혹시 어떤 방식으로 메모리 증가를 측정했을까요?
c#이 사용하는 managed가 관리하는 메모리는 GC에 의해서 관리되기에
native메모리의 증가/감소는 즉각적이지 않고, 대충 적당히 큰 힙 영역을 할당해서 부족할때마다 증가되는 방식이라서, memps같은 것으로는 증가 감소를 즉각적으로 확인하기는 어렵고, .net 메모리 프로파일러를 통해서나 볼수 있지만, struct의 경우 heap을 사용하지 않기에 여기에도 잡히지는 않거든요...